docs/source/about/release-notes.rst

```diff
@@ -43,7 +43,7 @@ v1.0.0
 * Upgraded to Hibernate Validator 5.2.4.Final
 * Upgraded to HSQLDB 2.3.4
 * Upgraded to Jadira Usertype Core 5.0.0.GA
-* Upgraded to Jackson 2.7.6
+* Upgraded to Jackson 2.8.1
```

Comment:
This should probably be moved to the `1.1.0` section, as `1.0.0` has already shipped ;)